packages/relay/src/lib/eth.ts

```diff
@@ -105,7 +107,25 @@ export class EthImpl implements Eth {
    */
   async feeHistory() {
     this.logger.trace('feeHistory()');
-    return await this.mirrorNode.getFeeHistory();
+    try {
+      const exchangeFileBytes = await (new FileContentsQuery()
```

Comment:
It is updated adhoc. Caching for 10 minutes or more should suffice.

Reply:
Actually, I thought it was fee schedule because I saw `feeHistory`. This is referring to the exchange rate file so it is updated once an hour, I think on the hour. The mirror node caches it for 60 seconds.

Is there a reason we're not using the new mirror node exchange rate REST API?

Reply:
We should be now.
I just implemented it in 0.57.0 so we were waiting for it to go out.
rc release has been cut so we can now code against it.
Will open a ticket for this